lib/active_record/base_without_table.rb

```diff
@@ -25,6 +25,10 @@ class BaseWithoutTable < Base
     self.abstract_class = true
 
     class << self
+      def new(args)
```

Comment:
shouldn't this be something like `*args`? otherwise not sure how multiple args get passed.